src/Features/Core/Portable/TodoComments/ITodoCommentService.cs

```diff
@@ -39,10 +27,35 @@ public TodoComment(TodoCommentDescriptor descriptor, string message, int positio
             Message = message;
             Position = position;
         }
+
+        internal TodoCommentInfo Convert(
+            Document document, SourceText text, SyntaxTree tree)
```

Comment:
>Document document, SourceText text, SyntaxTree tree [](start = 12, length = 51)

`Convert` isn't the best name for this. Perhaps `GetSerizableData`?

nit: this doesn't need to be on a separate line

Reply:
it goes off the end of my screen :)  so i wrap.

Reply:
(some of us work on small laptops :))

Reply:
The rest of the method have lines that are longer.

Reply:
Also, get a bigger laptop.